docs/content/TextInputWithTokens.mdx

```diff
@@ -213,12 +213,19 @@ render(MaxHeightExample)
     description="Whether tokens should render inline horizontally. By default, tokens wrap to new lines"
   />
   <PropsTable.Row name="size" type="TokenSizeKeys" defaultValue="extralarge" description="The size of the tokens" />
+  <PropsTable.Row
+    name="inputSize"
+    type="'small' | 'large'"
+    defaultValue="small"
+    description="The size of the text input"
+  />
   <PropsTable.Row
     name="hideTokenRemoveButtons"
     type="boolean"
     defaultValue="false"
     description="Whether the remove buttons should be rendered in the tokens"
   />
+  <PropsTableRow name="validationStatus" type="'warning' | 'error'" description="Style the input to match the status" />
```

Comment:
If we're not removing 'warning' as part of this issue, do we need to create another one or does it already exist?

Reply:
Think you're missing `'success'` from here too?

```suggestion
  <PropsTableRow name="validationStatus" type="'warning' | 'error' | 'success'" description="Style the input to match the status" />
```

Reply:
Another one docs related one (sorry): Can we add an example showing `validationStatus` being applied?

Reply:
> If we're not removing 'warning' as part of this issue, do we need to create another one or does it already exist?

We need to create one

Reply:
> Another one docs related one (sorry): Can we add an example showing validationStatus being applied?

Yes